examples/src/index.ts

```diff
@@ -23,7 +23,7 @@ async function main() {
   // await bootstrapPool({ account, asset_1, asset_2 });
 
   // Add some initial liquidity to the pool
-  await addInitialLiquidity({account, asset_1, asset_2});
+  // await addInitialLiquidity({account, asset_1, asset_2});
```

Comment:
I think we can uncomment line 23 and line 26 and leave comments above the lines like:
`// Comment the next line if you already bootstrapped the pool `

Reply:
It makes more sense 👍